db/migrate/20210111113140_add_vodafone_to_participating_mno_list.rb

```diff
@@ -0,0 +1,12 @@
+class AddVodafoneToParticipatingMnoList < ActiveRecord::Migration[6.0]
+  def change
+    reversible do |dir|
+      dir.up do
+        mno = MobileNetwork.find_by(brand: 'Vodafone')
+        mno.users.delete_all
```

Comment:
I'm not sure we'll want to remove them, but just `update(mobile_network_id: nil)`

Reply:
Does this need to be a migration?

Reply:
That should only remove the association but we could do this manually if you're happier with that

Reply:
Removed.

Reply:
The rest were done manually 🤷‍♂️